source/_data/SymbioticLab.bib

```diff
@@ -1549,3 +1549,20 @@ @InProceedings{oobleck:sosp23
     Oobleck enables resilient distributed training of large DNN models with guaranteed fault tolerance. It takes a planning-execution co-design approach, where it first generates a set of heterogeneous pipeline templates and instantiates at least f+1 logically equivalent pipeline replicas to tolerate any f simultaneous failures. During execution, it relies on already-replicated model states across the replicas to provide fast recovery. Oobleck provably guarantees that some combination of the initially created pipeline templates can be used to cover all available resources after f or fewer simultaneous failures, thereby avoiding resource idling at all times. Evaluation on large DNN models with billions of parameters shows that Oobleck provides consistently high throughput, and it outperforms state-of-the-art fault tolerance solutions like Bamboo and Varuna by up to 13.9x.
   }}
 
+
+
+@InProceedings{auxo:socc23,
+  author    = {Jiachen Liu, Fan Lai, Yinwei Dai, Aditya Akella, Harsha Madhyastha, Mosharaf Chowdhury},
```

Comment:
```suggestion
  author    = {Jiachen Liu and Fan Lai and Yinwei Dai and Aditya Akella and Harsha Madhyastha and Mosharaf Chowdhury},
```

Our plugin doesn't support comma-separated author list, rendering it as:
```
Yinwei Dai, Aditya Akella, Harsha Madhyastha, Mosharaf Chowdhury Jiachen Liu
```
Instead we need to use `and`-separated author list. Also my vscode complains that white spaces between authors are a special unicode U+00a0.

Reply:
Thank you for pointing out!